src/test/codegen/abi-main-signature-32bit-c-int.rs

```diff
@@ -7,4 +7,4 @@
 fn main() {
 }
 
-// CHECK: define i32 @main(i32{{( %0)?}}, i8**{{( %1)?}})
+// CHECK: define i32 @main(i32{{( %0)?}}, {{i8**|ptr}}{{( %1)?}})
```

Comment:
This probably needs `i8\*\*` since it's inside a regex, right? (Similar thing in other files)